src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx

```diff
@@ -234,15 +262,19 @@ const OrderDetailsPage: React.FC<OrderDetailsPageProps> = props => {
                     />
                   </React.Fragment>
                 ))}
-                <OrderPayment
-                  order={order}
-                  onCapture={onPaymentCapture}
-                  onMarkAsPaid={onPaymentPaid}
-                  onRefund={onPaymentRefund}
-                  onVoid={onPaymentVoid}
-                />
-                <CardSpacer />
-                <Metadata data={data} onChange={changeMetadata} />
+                {order?.status !== OrderStatus.UNCONFIRMED && (
```

Comment:
Since we're using this condition 3 times in this file, it'd be nice to put it in a named const, maybe `isOrderUnconfirmed` or so